enterprise/internal/own/service.go

```diff
@@ -133,6 +156,20 @@ func (s *service) ResolveOwnersWithType(ctx context.Context, protoOwners []*code
 	return resolved, nil
 }
 
+func (s *service) AssignedOwnership(ctx context.Context, repoID api.RepoID, _ api.CommitID) (AssignedOwners, error) {
+	summaries, err := s.db.AssignedOwners().ListAssignedOwnersForRepo(ctx, repoID)
+	if err != nil {
+		return nil, err
+	}
+	assignedOwners := AssignedOwners{}
+	for _, s := range summaries {
+		byPath := assignedOwners[s.FilePath]
+		byPath = append(byPath, *s)
+		assignedOwners[s.FilePath] = byPath
+	}
```

Comment:
I had a moment of confusion here about how it compiles if we're appending the service 😬 
```suggestion
	for _, summary := range summaries {
		byPath := assignedOwners[summary.FilePath]
		byPath = append(byPath, *summary)
		assignedOwners[summary.FilePath] = byPath
	}
```